src/vm/methodtable.cpp

```diff
@@ -9584,6 +9584,14 @@ BOOL MethodTable::ContainsGenericMethodVariables()
     return FALSE;
 }
 
+BOOL MethodTable::ContainsStackPtr() const
```

Comment:
This renamed to `IsByRefLike()` in CoreCLR. Delete, and use `IsByRefLike()` instead.